core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1898,6 +1898,11 @@ public void delete(AlluxioURI path, DeleteContext context)
           if (!childrenMountPoints.isEmpty()) {
             LOG.debug("Recursively deleting {} which contains mount points {}",
                 path, childrenMountPoints);
+            if (!context.getOptions().hasDeleteMountPoint()
+                || !context.getOptions().getDeleteMountPoint()) {
+              throw new AccessControlException("Cannot delete path {} which contains mount point "
+                  + "and without deleteMountPoint option specified to true");
```

Comment:
```suggestion
              auditContext.setAllowed(false);
              throw new AccessControlException(String.format("Cannot delete path %s which is or contains a mount point "
                  + "without deleteMountPoint option specified", path));
```

Reply:
@dbw9580 a typo in the sentence?

Reply:
Also in the exception message, add `you can delete mount points with -d or --deleteMountPoint`

Reply:
> @dbw9580 a typo in the sentence?

where is the typo?